instructure/canvas-ios
# [Teacher] Support modules with more than 100 items

refs: MBL-13507, MBL-13154
affects: teacher
release note: Fixed modules with more than 100 items

Test plan:
* See [ticket](https://instructure.atlassian.net/browse/MBL-13507) for a module with more than 100 items. All the items should load eventually as you scroll
* Use narmstrong > t and view CS 1400 > Modules
* Make sure all the modules and their items load eventually
* Collapse and expand module headers
* View CS 1400 > Pages > Links to Stuff > Module 1 > see that it auto scrolls to the module
* Tap the various module item types and see that they load
* See [this ticket as well](https://instructure.atlassian.net/browse/MBL-13154). I left a comment with login credentials for that account. Follow the repro in that ticket.
